dali/python/backend_impl.cc

```diff
@@ -454,14 +503,38 @@ void ExposeTensorList(py::module &m) {
       non_blocking : bool
             Asynchronous copy.
       )code")
-    .def("at",
+    .def("__getitem__",
         [](TensorList<GPUBackend> &t, Index id) -> std::unique_ptr<Tensor<GPUBackend>> {
```

Comment:
What happens if user calls `tl[-1]`? Will it go here or to the version with slice? AFAIK -1 is a valid index in Python and should mean "last element" or am I mistaken?